open-api/rest-catalog-open-api.yaml

```diff
@@ -134,9 +134,20 @@ paths:
     post:
       tags:
         - OAuth2 API
-      summary: Get a token using an OAuth2 flow
+      summary: Get a token using an OAuth2 flow (DEPRECATED for REMOVAL)
+      deprecated: true
       operationId: getToken
       description:
+        The `oauth/tokens` endpoint is **DEPRECATED for REMOVAL**. It is not recommended to _not_
```

Comment:
`It is not recommended to _not_ implement this endpoint` seems quite confusing. Did you mean `It is not recommended to implement this endpoint...`?